vars/mtaBuild.groovy

```diff
@@ -25,7 +25,7 @@ import static com.sap.piper.Utils.downloadSettingsFromUrl
     'extension',
     /**
      * The location of the SAP Multitarget Application Archive Builder jar file, including file name and extension.
-     * If it is not provided, the build tries to use Docker.
+     * If you run on Docker, this must match the location of the jar file in the container as well.
```

Comment:
Seems to be self-evident to me. And if we do not run in a container it must match the location in the local file system.

Reply:
This was not obvious to me :)

Reply:
Isn't it normal that that `mtaJarLocation` must match the location of the mta jar? What else should we expect? Now we document that this must match _when running via docker_. But it must match also when running not in docker.

Seriously: describing that an `mtaJarLocation` must match the location where the mta jar is located is from my point of view not an improvement. To my understanding the docu should be short and precise, maybe also providing some not obvious context. But the docu should not repeat evident things.

Another thing is: what we have currently

> If it is not provided, the build tries to use Docker.

is false. That needs to be removed. The decission about running in docker is made upon the corresponding docker image property